frontend/.storybook/main.js

```diff
@@ -1,13 +1,20 @@
+const webpack = require("webpack");
+
 module.exports = {
   stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
   addons: [
     "@storybook/addon-links",
-    "@storybook/addon-essentials",
-    "@storybook/addon-interactions",
     "storybook-addon-apollo-client",
     "@storybook/preset-create-react-app",
+    "@storybook/addon-docs",
   ],
   webpackFinal: async (config) => {
+    config.plugins.push(
+      new webpack.ProvidePlugin({
+        process: "process/browser",
```

Comment:
fixes issue where build was failing due to `process` not being defined. open to other ways to fix this